manifests/vc/standalone_test.pp

```diff
@@ -110,6 +111,23 @@
     content => template("sunet/vc/standalone/test/Makefile.erb")
   }
 
+  schedule { 'update_letsencrypt_time':
+    period => monthly,
+    repeat => 1,
+  }
+
+  exec { 'install_letsencrypt':
```

Comment:
packages should be installed with; `package { '<package>': ensure => 'installed' }`